Arcade 7.0 upgrade breaks BuildIBCTrainingSettings

<!-- If these statements apply, replace [ ] with [x] before filing your issue. -->
- [x] This issue is blocking MSBuild's upgrade to Arcade 7 (which may block sourcebuild for RC2, not clear to me yet)
- [ ] This issue is causing unreasonable pain

I am updating MSBuild to Arcade 7 in dotnet/msbuild#7990, but am currently blocked on this official build failure:

https://dev.azure.com/devdiv/DevDiv/_build/results?buildId=6712662&view=logs&j=bb592630-4b9d-53ad-3960-d954a70a95cf&t=fde09a22-276d-5f89-3cea-360c350845ce&l=18

```
##[error].packages\microsoft.dotnet.arcade.sdk\7.0.0-beta.22464.4\tools\SdkTasks\VisualStudio.BuildIbcTrainingSettings.proj(20,3): error MSB4019: The imported project "D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\7.0.0-beta.22464.4\tools\SdkTasks\Directory.Build.targets" was not found. Confirm that the expression in the Import declaration "Directory.Build.targets" is correct, and that the file exists on disk.
```

I suspect this arose after https://github.com/dotnet/arcade/pull/9373 (cc @ViktorHofer) deleted the `Directory.Build.targets` file. Deleting this line locally avoids the problem (though quickly fails later because I'm not running the real official build).

https://github.com/dotnet/arcade/blob/720af493900b2f2bdc48e9ee12577983a5c9be36/src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/VisualStudio.BuildIbcTrainingSettings.proj#L20

--- a/src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/VisualStudio.BuildIbcTrainingSettings.proj
+++ b/src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/VisualStudio.BuildIbcTrainingSettings.proj
@@ -17,7 +17,6 @@
   -->
 
   <Import Project="Directory.Build.props" />
-  <Import Project="Directory.Build.targets" />
 
   <PropertyGroup>
     <_VisualStudioBuildTasksAssembly>$(NuGetPackageRoot)microsoft.dotnet.build.tasks.visualstudio\$(MicrosoftDotNetBuildTasksVisualStudioVersion)\tools\net472\Microsoft.DotNet.Build.Tasks.VisualStudio.dll</_VisualStudioBuildTasksAssembly>

